src/porepy/numerics/ad/operators.py

```diff
@@ -1332,11 +1332,11 @@ class TimeDependentDenseArray(Operator):
     def __init__(
         self,
         name: str,
-        domains: GridsLike,
+        domains: GridLikeSequence,
```

Comment:
Should we generalize this signature to the base class Operator?
With the property _grid_type. 
All children of operator can then have a simplified check of grid_type in the implementations of their methods.
But also other objects operating on instances of Operator.

Reply:
I don't understand exactly what you mean by generalizing the signature. Could you elaborate, please?

Reply:
@vlipovac do you propose to use `GridLikeSequence` for the base class Operator, and move `_grid_type` to the base class? If so, I agree

Reply:
@IvarStefansson @Yuriyzabegaev exactly like Yuriy summarized it.
class Operator currently has interfaces and subdomains as arguments in the constructor.
It may be more general (and consistent with derived classes) to use the same approach as here.

Reply:
I like the idea. I think we need @keileg's opinion as well, though.

Reply:
Sounds good.